saleor/csv/models.py

```diff
@@ -0,0 +1,14 @@
+from django.db import models
+
+from ..account.models import User
+from . import JobStatus
+
+
+class Job(models.Model):
+    status = models.CharField(
+        max_length=50, choices=JobStatus.CHOICES, default=JobStatus.PENDING
+    )
+    user = models.ForeignKey(User, related_name="jobs", on_delete=models.CASCADE)
+    content_file = models.FileField(upload_to="csv_files", null=True)
+    created_at = models.DateTimeField(auto_now_add=True)
+    ended_at = models.DateTimeField(null=True)
```

Comment:
It sounds weird to me. `completed_at` maybe?

Reply:
Yeah, `completed_at` sounds better.